ArticleTemplates/assets/js/modules/util.js

```diff
@@ -41,6 +41,34 @@ define(function () {
 
         isOnline: function () {
             return !document.body.classList.contains('offline') && navigator.onLine;
+        },
```

Comment:
I also wonder whether it might be worth using something like underscore but enabling you to import only the methods you use (like we do on dotcom), that would probably give you a lot of util methods (though a few possibly superseded by ES2015 if you end up using that...)